WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingBottomSheetFragment.kt

```diff
@@ -1,28 +1,39 @@
 package org.wordpress.android.ui.posts
 
-import android.content.DialogInterface
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.FrameLayout
+import androidx.annotation.NonNull
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.ViewModelProviders
 import com.google.android.material.bottomsheet.BottomSheetBehavior
 import com.google.android.material.bottomsheet.BottomSheetDialog
 import org.wordpress.android.R
 import org.wordpress.android.WordPress
+import org.wordpress.android.fluxc.model.SiteModel
 import org.wordpress.android.login.widgets.WPBottomSheetDialogFragment
+import org.wordpress.android.ui.posts.ActionState.TagsActionState
+import org.wordpress.android.ui.posts.PrepublishingScreen.HOME
 import javax.inject.Inject
 
-class PrepublishingBottomSheetFragment : WPBottomSheetDialogFragment() {
+class PrepublishingBottomSheetFragment : WPBottomSheetDialogFragment(), TagsSelectedListener {
     @Inject internal lateinit var viewModelFactory: ViewModelProvider.Factory
 
-    private lateinit var viewModel: PrepublishingViewModel
+    private lateinit var prepublishingViewModel: PrepublishingViewModel
+    private lateinit var prepublishingActionsViewModel: PrepublishingActionsViewModel
+
+    private lateinit var site: SiteModel
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        setStyle(DialogFragment.STYLE_NORMAL, R.style.WordPress_BottomSheetDialogTheme)
```

Comment:
Could we set this in the layout or it's not possible?

Reply:
I just realized this isn't needed here right now. I placed it here because I was going to use a custom theme but didn't and added the new attribute to our base style but as you pointed it, this change can impact other bottom sheets, and since this change will cause the bottom sheet to readjust based on the keyboard I am thinking to just make a custom style for this instead of modifying the base. This comment allowed me to take that approach. I appreciate it.